R/functions__eager.R

```diff
@@ -133,7 +135,8 @@ pl$date_range = function(
     closed = "both", # : ClosedInterval = "both",
     name = NULL, # : str | None = None,
     time_unit = "us",
-    time_zone = NULL # : str | None = None
+    time_zone = NULL, # : str | None = None
+    explode = TRUE
```

Comment:
this arg is not in [py-polars](https://pola-rs.github.io/polars/py-polars/html/reference/expressions/api/polars.date_range.html#polars.date_range). Is there a reason to add it? I guess the user can do `pl$date_range()$explode()` instead?

Reply:
my bad, didn't see this point in your original post